#Intel compiler does not understand __has_attribute(attribute
# Oct 24th 2016 by B. Hajgato (Free University Brussels - VUB)
--- GraphicsMagick-1.3.25/magick/common.h.orig	2016-09-05 21:20:24.000000000 +0200
+++ GraphicsMagick-1.3.25/magick/common.h	2016-10-24 15:03:21.858939448 +0200
@@ -128,7 +128,7 @@
 #    define MAGICK_ATTRIBUTE(x) /*nothing*/
 #  else
 #    define MAGICK_ATTRIBUTE(x) __attribute__(x)
-#    if (defined(__clang__) || (defined(__GNUC__) && __GNUC__ >= 5))
+#    if (defined(__clang__) || ((defined(__GNUC__) && __GNUC__ >= 5) && !__INTEL_COMPILER))
 #      define MAGICK_HAS_ATTRIBUTE(attribute) __has_attribute(attribute)
 #    else
 #      define MAGICK_HAS_ATTRIBUTE(attribute) (0)
